lib/puppet/indirector/catalog/rest.rb

```diff
@@ -13,6 +13,14 @@ def find(request)
 
     session = Puppet.lookup(:http_session)
     api = session.route_to(:puppet)
+
+    ip_address = begin
+      " (#{Resolv.new.getaddress(api.url.host)})"
+    rescue Resolv::ResolvError
+      nil
+    end
+    Puppet.notice("Requesting catalog from #{URI(api.url).host}:#{URI(api.url).port}#{ip_address}")
```

Comment:
I think we can grab the host/port from the url reader directly like

```suggestion
    Puppet.notice("Requesting catalog from #{api.url.host}:#{api.url.port}#{ip_address}")
```